### PR TITLE
Enable vacuum on all tables

### PR DESCRIPTION
**Pull request by sethvargo**
Fixes GH-289

/assign @mikehelmick

### CODE REVIEW

File: migrations/000017_vacuum.down.sql [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TABLE APIConfig SET (autovacuum_enabled = false);
+ALTER TABLE ExportBatch SET (autovacuum_enabled = false);
+ALTER TABLE ExportConfig SET (autovacuum_enabled = false);
+ALTER TABLE ExportFile SET (autovacuum_enabled = false);
+ALTER TABLE Exposure SET (autovacuum_enabled = false);
```
Comment (mikehelmick) [inline migrations/000017_vacuum.down.sql:21]: I think we need to tune this param lower

https://www.postgresql.org/docs/8.4/runtime-config-client.html#GUC-VACUUM-FREEZE-TABLE-AGE

the default is to wait for up to 150 million transactions - which is many many days

Comment (mikehelmick) [reply migrations/000017_vacuum.down.sql:21]: perhaps others. - @jba - do you have any thoughts?

Comment (jba) [reply migrations/000017_vacuum.down.sql:21]: We seem to use whatever defaults GCP gives us. The only things we change are
```
    {
      name  = "autovacuum_vacuum_cost_limit"
      value = 10000
    },
    {
      name  = "vacuum_cost_limit"
      value = 10000
    }
```

Comment (sethvargo) [reply migrations/000017_vacuum.down.sql:21]: Vacuuming is _very_ expensive and acquires a full table lock. Do we want to do it more frequently?

**Pull request merged**